bodhi/server/models.py

```diff
@@ -1149,6 +1153,10 @@ class Update(Base):
 
     user_id = Column(Integer, ForeignKey('users.id'))
 
+    # Greenwave
+    greenwave_policies_satisfied = Column(Boolean, default=True, nullable=False)
```

Comment:
@matt8754 Greenwave can tell about the "final" states though - "passed", "failed", and "skipped". I think we should get this patch to use the ```ci_status``` field (we can rename it to ```test_gating_status``` or something if you like that better, to be consistent with the setting name), but this patch can just worry about those three states.

I can write more code later that can figure out how to do "waiting" and "running" and "queued" and make that a separate concern. Those states are really just for user convenience/feedback and aren't necessary for Bodhi to function correctly. As long as we can reliably land in "passed", "failed", or "skipped" then I think this patch will be good to go, and Greenwave is able to tell us about those states.